source/blog/2016-01-07-atomic-diff.html.md

```diff
@@ -0,0 +1,136 @@
+---
+title: atomic diff
+author: baude
+date: 2016-01-07 20:34:42 UTC
+published: false
+comments: true
+---
+
+The original mission of the [atomic application](https://github.com/projectatomic/atomic) to install, manage, and run container images using container Labels, has slowly been growing.  We have been concentrating on ease of use and value-add functions for containers and images. We recently added the `atomic diff` command.  This command allows you to differentiate between two container images. This is critically important if you are like me and you have hundreds of container images on your system. I also use it when I pull an updated image from a repository and want to see what has changed.
```

Comment:
"and run container images using container Labels,": does it have to be capitalized here? Also I feel like dropping the second "container" would flow better.